release-builder.sh

```diff
@@ -50,25 +49,159 @@ STATIC_CONTAINER=$(docker ps -a | grep $CONTAINER_NAME |wc -l)
 if false; then
   echo "Static container, execute in static container to have max. cache"
   docker start $CONTAINER_NAME
-  docker exec -i $CONTAINER_NAME /hbb_exe/activate-exec bash -x /io/build-core.sh "$GITHUB_REPOSITORY" "$GITHUB_SHA" "$BUILD_CORES" "$GITHUB_RUN_NUMBER"
+  docker exec -i $CONTAINER_NAME /hbb_exe/activate-exec bash -c "source /opt/rh/gcc-toolset-11/enable && bash -x /io/build-core.sh '$GITHUB_REPOSITORY' '$GITHUB_SHA' '$BUILD_CORES' '$GITHUB_RUN_NUMBER'"
   docker stop $CONTAINER_NAME
 else
   echo "No static container, build on temp container"
   rm -rf release-build;
   mkdir -p release-build;
 
+  docker volume create cache-volume
```

Comment:
If this is running on an actions runner with persistent state, which I believe the self-hosted runners qualify as, if there's no volume it will succeed but if it's already existing it will error and the build will stop (`set -ex`) 

It may also pay to have a more descriptive name, "gh-actions-conan-ccache-cache" (or whatever takes your fancy) given the machines may be used for other projects?

Reply:
If the volume already exists, it will be ignored.

https://docs.docker.com/reference/cli/docker/volume/create/#:~:text=If%20you%20specify%20a%20volume%20name%20already%20in%20use%20on%20the%20current%20driver%2C%20Docker%20assumes%20you%20want%20to%20reuse%20the%20existing%20volume%20and%20doesn%27t%20return%20an%20error.

There is no objection to using a different name for the volume name. Considering that it will also be used locally, how about something like `xahau-release-builder-cache`?

Reply:
> If the volume already exists, it will be ignored.

Oh :) 

> Considering that it will also be used locally, how about something like xahau-release-builder-cache?

works for me